userspace/libsinsp/filter/ast.h

```diff
@@ -225,9 +265,12 @@ struct SINSP_PUBLIC not_expr: expr
 
     std::unique_ptr<expr> child;
 
-    static std::unique_ptr<not_expr> create(std::unique_ptr<expr> c)
+    static std::unique_ptr<not_expr> create(std::unique_ptr<expr> c,
+					    const libsinsp::filter::ast::pos_info& pos = s_initial_pos)
     {
-        return std::unique_ptr<not_expr>(new not_expr(std::move(c)));
+        std::unique_ptr<not_expr> ret(new not_expr(std::move(c)));
+	ret->set_pos(pos);
+	return ret;
```

Comment:
This is an ultra nitpick and it should be addressed when we enforce code formatting, but I see some changes mix up tabs and space for indentations, would you mind unifying them into spaces?

Reply:
We're kind of all over the map wrt spaces vs tabs though. The majority of the libsinsp code is tabs. Even many of the other files below libsinsp/filter are using tabs.

I'd rather we tackle formatting stuff like this in one change by adding a formatter as a commit hook (or whatever else is best practice).